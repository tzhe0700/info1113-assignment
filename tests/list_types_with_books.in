ADD COLLECTION spellbooks.csv
LIST TYPES
EXIT
