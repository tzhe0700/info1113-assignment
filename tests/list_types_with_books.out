17 spellbooks successfully added.
Charm
Counter-spell
Curse
Healing
Hex
Jinx
Transfiguration

